targets/targets.json

```diff
@@ -5116,15 +5117,16 @@
             "SERIAL",
             "SPI",
             "SPISLAVE",
-            "TSC",
-            "MPU"
+            "TSC"
```

Comment:
I'm curious. What is TSC?

Reply:
It is Touch Screen. This change is just to delete the "MPU", but I have to delete the comma behind "TSC".